README.md

```diff
@@ -1 +1,73 @@
-# edge-api-utils
\ No newline at end of file
+# edge-api-utils
+
+Copyright (C) 2021 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License,
+Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
+
+## Introduction
+The purpose of this edge API utils is the combine classes/utilities/constants shared by both modules: edge-common and edge-common-spring.
+
+## Additional information
+
+
+### Secure Stores
+
+Three secure stores currently implemented for safe retrieval of encrypted credentials:
+
+#### EphemeralStore ####
+
+Only intended for _development purposes_.  Credentials are defined in plain text in a specified properties file.  `src/main/resources/ephemeral.properties`
+
+#### AwsParamStore ####
+
+Retrieves credentials from Amazon Web Services Systems Manager (AWS SSM), more specifically the Parameter Store, where they're stored encrypted using a KMS key.  `src.main/resources/aws_ss.properties`
+
+**Key:** `<salt>_<tenantId>_<username>`
+
+e.g. Key=`ab73kbw90e_diku_diku`
+
+#### VaultStore ####
+
+Retrieves credentials from a Vault (https://vaultproject.io).  This was added as a more generic alternative for those not using AWS.  `src/main/resources/vault.properties`
+
+**Key:** `<salt>/<tenantId>`
+**Field:** `<username>`
+
+e.g. Key=`ab73kbw90e/diku`, Field=`diku`
+
+## Configuration
+
+Configuration information is specified in two forms:
+1. System Properties - General configuration
+1. Properties File - Configuration specific to the desired secure store
+
+### System Properties
```

Comment:
this section doesn't belong here - this is a library of utilities; a build-time dependency.

Reply:
Removed